bin/mev/src/cmd/build.rs

```diff
@@ -16,4 +16,4 @@ impl TryFrom<CliArgs> for Config {
     }
 }
 
-pub type Command = Cli<CliArgs>;
+pub type Command = Cli<EthereumChainSpecParser, CliArgs>;
```

Comment:
it looks like we can't do any type inference here, e.g. `Cli<_, CliArgs>;`, which would let us skip the import here

I tried a demo and it looks like rust doesn't allow placeholders on type aliases